File: app-components/src/externalTesting/java/org/synsystems/onlypass/components/ExternalTestingApp.java
import org.synsystems.onlypass.components.App;
import org.synsystems.onlypass.components.AppComponent;

import org.synsystems.onlypass.components.DaggerExternalTestingAppComponent;

public class ExternalTestingApp extends App {
  @Override
  protected AppComponent createAppComponent() {
    return DaggerExternalTestingAppComponent.create();
  }
}

File: app-components/src/externalTesting/java/org/synsystems/onlypass/components/ExternalTestingApp.kt
import org.synsystems.onlypass.components.App
import org.synsystems.onlypass.components.AppComponent
import org.synsystems.onlypass.components.DaggerExternalTestingAppComponent

class ExternalTestingApp : App() {

  @Override
  protected fun createAppComponent() = DaggerExternalTestingAppComponent.create()
}